src/content/profile-data.js

```diff
@@ -269,7 +267,7 @@ export function collapsePlatformStackFrames(thread: Thread) {
               // Create a new platform frame
               const newFuncIndex = newFuncTable.length++;
               newFuncTable.name.push(stringTable.indexForString('Platform'));
-              newFuncTable.resource.push(null);
+              newFuncTable.resource.push(-1);
```

Comment:
Flow caught this bug of mine. I kept it all `-1` until we land your other patch.